zerver/lib/logging_util.py

```diff
@@ -177,11 +177,17 @@ class ZulipFormatter(logging.Formatter):
     # Used in the base implementation.  Default uses `,`.
     default_msec_format = '%s.%03d'
 
-    _fmt = '%(asctime)s %(zulip_level_abbrev)-4s [%(zulip_origin)s] %(message)s'
-
     def __init__(self):
         # type: () -> None
-        super().__init__(fmt=self._fmt)
+        super().__init__(fmt=self._fmt())
+
+    def _fmt(self):
```

Comment:
`def _fmt(self) -> str:` ? :>

Reply:
I'd rather spend as little time as possible in the state where we have two different styles -- so once we have a plan that will convert everything in short order, and we've begun to do that conversion, then it'll be good to start writing in the new style.